promql/functions.go

```diff
@@ -765,6 +807,13 @@ var functions = map[string]*Function{
 		ReturnType: model.ValVector,
 		Call:       funcLabelReplace,
 	},
+	"label_join": {
+		Name:       "label_join",
+		ArgTypes:   []model.ValueType{model.ValVector, model.ValString, model.ValString, model.ValString, model.ValString, model.ValString},
```

Comment:
`ArgTypes` should only have 4 values since its variadic. However, I get `unexpected error` when I make it 4 values instead of 6.

@fabxc : Any thoughts?